app/views/partners/show.html.erb

```diff
@@ -90,6 +90,36 @@
           </div>
         </section>
 
+        <section class="card card-info card-outline" id="partner-users">
+          <div class="card-header">
+            <h2 class="card-title">Users who can access this Partner</h2>
+          </div>
+          <div class="card-body p-0">
+            <div class="tab-content" id="custom-tabs-three-tabContent">
+              <table class="table">
+                <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Email</th>
+                  <th>Invitation Sent</th>
+                  <th>Last Logged In</th>
+                </tr>
+                </thead>
+                <tbody>
+                <% @partner_users.each do |user| %>
+                  <tr>
+                    <td><%= user.name %></td>
+                    <td><%= user.email %></td>
+                    <td><%= user.invitation_sent_at && user.invitation_sent_at.strftime('%B %-d, %Y') %></td>
+                    <td><%= user.last_sign_in_at && user.last_sign_in_at.strftime('%B %-d, %Y') %></td>
```

Comment:
Minor - we could use the locale helper to format the times for us, that way they are consistent across the app

```suggestion
                    <td><%= user.invitation_sent_at && l user.invitation_sent_at %></td>
                    <td><%= user.last_sign_in_at && l user.last_sign_in_at %></td>
```

Reply:
Is there any alias for the `l` helper @jhuizy that might be clearer on what it does? I worry that it isn't entirely clear for someone who doesn't know about the helper to figure out what it is being added for.

Reply:
`:l` is an alias for `:localize`. I'd prefer the latter since I default to spelling things out when feasible.